loaders/src/glTF/2.0/Extensions/EXT_texture_webp.ts

```diff
@@ -0,0 +1,47 @@
+import { IGLTFLoaderExtension } from "../glTFLoaderExtension";
+import { GLTFLoader, ArrayItem } from "../glTFLoader";
+import { ITexture } from "../glTFLoaderInterfaces";
+import { BaseTexture } from "babylonjs/Materials/Textures/baseTexture";
+import { Nullable } from "babylonjs/types";
+import { IKHRTextureBasisU } from 'babylonjs-gltf2interface';
+
+const NAME = "EXT_texture_webp";
+
+/**
+ * [Proposed Specification](https://github.com/KhronosGroup/glTF/pull/1534)
+ * !!! Experimental Extension Subject to Changes !!!
+ */

```

Comment:
As I can see it, the extension is now live on Khronos side (https://github.com/KhronosGroup/glTF/blob/master/extensions/README.md)

Reply:
oh yeah..even better ;D